docs/guides/12-add-widgets-advanced.md

```diff
@@ -0,0 +1,262 @@
+## Add widgets advanced
+
+We have seen how to add legends to our map by using `getLegendsData` method, and we have also learned how to display widgets using histogram expressions in the [previous guide](/developers/carto-vl/guides/add-widgets/). This is an advanced guide to learn how to obtain and display the information in widgets by using `viewportHistogram` and `globalHistogram` expressions.
+
+### Overview
+
+We will learn how to create bar charts for **categorical** data and histograms for **numeric** data. For this purpose, the following examples make use of the same source, the [**Vancouver Trees**](https://team.carto.com/u/cartovl/tables/cartovl.vancouver_trees/public/map) dataset.
```

Comment:
Nice dataset. But I have seen that it has some wrong features, in our beloved [Null Island](https://en.wikipedia.org/wiki/Null_Island) at (0, 0), so we should probably delete those.